README.md

```diff
@@ -1 +1,92 @@
 # Dictionary Component
+## Table of Contents
+* [General information](#general-information)
+* [Credentials](#credentials)
+* [Triggers](#triggers)
+* [Actions](#actions)
+   * [Action Name](#action-name)
+     * [Action Name. Config fields](#action-name-config-fields)
+* [Additional info](#additional-info)
+* [Known Limitations](#known-limitations)
+
+## General Information
+The Dictionary Component can be used to convert from different dictionaries that parsed as a CSV.
+
+## Credentials
+### CSV
+The CSV is entered on the Credentials page as a list of separated items. Any delimiter is supported. The CSV will be parsed using the first row as a header for each column below it. The CSV must be able to be interpreted as a rectangle, i.e. it cannot be missing values.
+
+For example,
+
+```
+English,Abbreviated,German
+male,M,männlich
+female,F,weiblich
+other,O,divers
+unknown,U,unbekannt
+```
+
+Will parse as
+
+| English | Abbreviated | German    | 
+|---------|-------------|-----------| 
+| male    | M           | männlich  | 
+| female  | F           | weiblich  | 
+| other   | O           | divers    | 
+| unknown | U           | unbekannt |
+
+and
+
+```
+English,Abbreviated,German
+male,M,männlich
+,F,weiblich
+other,,
+unknown,U,unbekannt
+```
+
+Will parse correctly as
+
+| English | Abbreviated | German    | 
+|---------|-------------|-----------| 
+| male    | M           | männlich  | 
+|         | F           | weiblich  | 
+| other   |             |           | 
+| unknown | U           | unbekannt | 
+
+(though this may not be useful), but
+
+```
+English,Abbreviated,German
+male,M,männlich
+F,weiblich
+other
+unknown,U,unbekannt
+```
+will provide a failed result.
+
+The CSV can only be a maximum of 5kB, and if it contains any duplicate values in a given column, it will fail validation.
+
+## Triggers
```

Comment:
We don't have triggers yes

Reply:
At one point I was told to include the sections even if there is nothing in them, for sake of coherency.